subjects/0-shell/README.md

```diff
@@ -0,0 +1,81 @@
+## 0-shell
+
+### Objective
+
+The objective of this project is for you to create a simple [shell](https://en.wikipedia.org/wiki/Shell_script).
+
+Through the `0-shell` you will get to the core of the `Unix` system and explore an important part of this system’s API witch is the process creation and synchronization.
+Executing a command inside a shell implies creating a new process, which execution and final state will be monitored by its parents process. This set of functions will be the key to success for your project.
+
+For this project you will only have to create a simple `Unix` interpreter where you can run some of the most known commands. For this part of the project, no advanced functions, pipes or redirection will be asked, you can add them if you like.
+
+### Instructions
+
+- You must program a mini `Unix` interpreter.
+- This interpreter must display at least a simple `$` and wait until you type a command line witch will be validated by pressing enter.
+- The `$` will be shown again only once the command has been completely executed.
+- The command lines are simple, you will not have pipes, redirection or any other advanced functions.
+- You must manage the errors, by displaying a message adapted to the error output.
+- You must implement the following commands that will behave exactly like the originals:
```

Comment:
There are no originals unfortunately.
So maybe we can advise to target a simple and widespread thing, like BusyBox.